class/defaults.yml

```diff
@@ -1,6 +1,15 @@
 parameters:
   openshift4_ingress:
-    namespace: openshift-ingress-operator
+    namespace: openshift-ingress
+    cloud:
+      provider: ${cloud:provider}
+      aws:
+        accessKey: copy-from-cluster
+      azure:
+        clientID: copy-from-cluster
```

Comment:
Are those dummy values meant to stay?

Reply:
I'd say so, yes. The workflow would be to compile with dummy vars and then, once the CredentialRequest is handled on the cluster, update with the correct values (as described in docs).

Reply:
I do not like but I do not have a suggestion how to do better.